tests/failpoints/cases/test_merge.rs

```diff
@@ -786,3 +785,76 @@ fn test_merge_cascade_merge_with_apply_yield() {
         cluster.must_put(format!("k{}", i).as_bytes(), b"v3");
     }
 }
+
+// Test if the rollback merge proposal is proposed before the majority of peers want to rollback
+#[test]
+fn test_node_mutiple_rollback_merge() {
+    let mut cluster = new_node_cluster(0, 3);
```

Comment:
I think you still need to use `crate::setup()`.

Reply:
Done